content/posts/2025/01/teaching-llms-report.md

```diff
@@ -0,0 +1,98 @@
+Instead, just as Carpentries workshops aim to instil good (enough) practices in software development and data organisation/handling, the majority of session participants saw Carpentries workshops as an opportunity for Instructors to teach good practices and help learners develop a better, safer, more useful understanding of LLMs.
+
+Furthermore, some participants asserted that usage of LLM assistants is increasingly widespread in the workplace and will likely become an expected skill in the coming years. 
+They felt that we would be doing learners a disservice by not teaching them the skills and knowledge to use these tools as part of their routine approach to software development and data analysis.
+
+However, participants also identified several potential downsides to formally introducing content on LLM assistants into the curriculum for Carpentries workshops. 
+First, that LLM tools may be unavailable or forbidden at some institutions, for example where usage is restricted to a specific platform on data privacy/regulatory grounds.
+This could make it difficult to develop content, examples, etc that are specific to a particular tool with confidence that it will be usable at workshops in all locations.
+Similarly, participants highlighted the potentially significant differences in output and performance from paid and free versions of the tools: that all the software we teach is free of cost is one of the major advantages to The Carpentries workshops, ensuring they are accessible to all learners. 
+But here we may find that free versions of LLM assistants are more limited in their potential usefulness to learners. 
+Furthermore, several participants sounded a note of caution around the currently free versions of the most popular tools, citing the danger that their owners may introduce fees in the future that could restrict Instructors' ability to teach lesson content about them in workshops.
+
+Finally, participants acknowledged that the topic would need a significant chunk of time dedicated to teaching and discussing it properly in a workshop -- time that cannot then be used for something else. 
+Most participants were Carpentries Instructors, familiar with the challenge of fitting the current lesson content into the time available at a two-day workshop. 
+How would we make space to teach LLM assistants effectively in workshops that are already packed to bursting with other important content?
+
+
+## LLM assistants and The Carpentries Core Values
+Central to the discussion in both community sessions was the question of how teaching LLMs in Carpentries workshops might align with or against [the community's Core Values](/about-us/#our-values). 
+Participants highlighted concerns about the lack of attribution of the data used to train the models (conflicting with our commitments to act openly and value all contributions), and the significant energy and resource requirements leading to environmental impact of the training process itself. 
```

Comment:
@OscarSiba - could you please provide guidance on whether our core values should be capitalized? I've most often seen them that way but am never sure.